WolvenKit.App/Controllers/RED4Controller.cs

```diff
@@ -463,6 +475,33 @@ private bool PackArchives(Cp77Project cp77Proj, LaunchProfile options)
 
             return true;
         }
+
+
+        private static bool PackScriptFiles(Cp77Project cp77Proj)
+        {
+            //preliminary approach -- everything under resources, except for *.yaml, *.xl, is a 'script' element
```

Comment:
hmm, this doesn't seem right? Scripts should only be ".reds" files... I know it's preliminary, but I think this is confusing. You could change the name of the function to "PackResources" I suppose.

Reply:
Although I have concerns about Scripts == `*.reds` exclusively, I do agree that this preliminary approach packs resources. I will change the method to "PackResources" so we have a path for users (like me) to deploy their scripts :)

The change will be made.

Reply:
Changes completed!  Please review.